docs/guides/use_checkpointing.md

```diff
@@ -23,45 +26,60 @@ This guide covers the following:
 * How to save and load checkpoints when you run in multi-host scenarios with
 [`flax.training.checkpoints.save_checkpoint_multiprocess`](https://flax.readthedocs.io/en/latest/api_reference/flax.training.html#flax.training.checkpoints.save_checkpoint_multiprocess).
 
+
 ## Setup
 
 Install/upgrade Flax, JAX, [Optax](https://optax.readthedocs.io/) and [TensorStore](https://google.github.io/tensorstore/). For JAX installation with GPU/TPU support, visit [this section on GitHub](https://github.com/google/jax#installation).
 
-```{code-cell} ipython3
-!pip install -U -q flax jax jaxlib optax tensorstore
+```python
+!pip install -U -qq flax jax jaxlib optax tensorstore orbax
```

Comment:
If the `flax` package contains JAX, Optax, etc, are we instructing users to _then upgrade_ `jax`/`jaxlib` and `optax`? Otherwise, maybe `!pip install -U -q flax tensorstore orbax` might be enough?

Reply:
Done.